dev/io.openliberty.session.6.0.internal/resources/io/openliberty/session60/internal/resources/SessionMessages.nlsprops

```diff
@@ -0,0 +1,30 @@
+###############################################################################
+# Copyright (c) 2022 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#CMVCPATHNAME io.openliberty.session.6.0.internal/resources/io/openliberty/session60/internal/resources/SessionMessages.nlsprops
+#COMPONENTPREFIX SESN
+#COMPONENTNAMEFOR SESN Session
+#ISMESSAGEFILE TRUE
+#NLS_MESSAGEFORMAT_VAR
+#NLS_ENCODING=UNICODE
+
+# Note:
+#Last used error meessage code was SESN8502I
+#
+#Use 8600 - 8699 for Servlet 6.0 - EE10
+#
+# -----------------------------------------------
+cookie.attribute.name.null=SESN8600E: The attribute name is null.
```

Comment:
Is the null attribute associated with the Cookie class? We want to give the user enough information so that they can locate the problem. How about something like "An attribute name on the Cookie class is null.".